resources/js/components/Locations/Charts/Download/Download.vue

```diff
@@ -50,11 +50,22 @@ export default {
                 locationId: this.locationId,
                 email: this.email
             });
+            this.email = '';
+            this.emailEntered = false;
         },
 
         textEntered()
         {
-            this.emailEntered = true;
+            let regexEmail = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/
```

Comment:
When the variable does not change, in javascript, use `const` as a constant. It let's the reader know this variable won't change